README.md

```diff
@@ -333,13 +333,21 @@ directly using webpack, for example:
 # .eslintrc.yml
 settings:
   import/parsers:
-    typescript-eslint-parser: [ .ts, .tsx ]
+    @typescript-eslint/parser: [ .ts, .tsx ]
```

Comment:
I don't think this is valid YAML. It gets confused because of the `@`, so you would have to make sure this key is a string by doing:
```suggestion
    '@typescript-eslint/parser': [ .ts, .tsx ]
```